distributed/utils.py

```diff
@@ -1368,3 +1369,33 @@ def is_valid_xml(text):
 async def offload(fn, *args, **kwargs):
     loop = asyncio.get_event_loop()
     return await loop.run_in_executor(_offload_executor, fn, *args, **kwargs)
+
+
+def serialize_for_cli(data):
+    """ Serialize data into a string that can be passthrough cli
+
+    Parameters
+    ----------
+    data: json-serializable object
+        The data to serialize
+    Returns
+    -------
+    serialized_data: str
+        The serialized data as a string
+    """
+    return base64.urlsafe_b64encode(json.dumps(data).encode()).decode()
+
+
+def deserialize_for_cli(data):
+    """ De-serialize data into the original object
+
+    Parameters
+    ----------
+    data: str
+        String serialied by serialize_for_cli()
+    Returns
+    -------
+    deserialized_data: obj
+        The de-serialized data
+    """
+    return json.loads(base64.urlsafe_b64decode(data.encode()).decode())
```

Comment:
It's unfortunate that we have to encode this here.  I guess that this is because we have to use `" ".join(...)`.  I took a quick look into `asyncssh` and indeed it seems to only accept a string, rather than a list of strings.

Reply:
I guess this feels especially odd because no user is ever going to actually use this command line keyword.  It's much more likely that they would use this with some JSON string instead.  

I wonder if perhaps there is some other way of passing this information that might be outside of the public API.  That way we wouldn't have to worry about breaking this in the future if we find something nicer.  Maybe environment variables?

Honestly, if you don't need SSH support right now I would be in favor of rolling back these changes, and just supporting LocalCluster for now.  SSHCluster is infrequently used, and not something that we necessarily need to slow things down over.

Reply:
I think we need ssh in `dask-cuda` to support a full DGX

Reply:
But I agree, there is no need to add a new public cli argument. 
I pass the config as the environment variable `DASK_INTERNAL_INHERIT_CONFIG` now.
Is there a naming scheme for internal  environment variables or is `DASK_INTERNAL_INHERIT_CONFIG` fine?

Reply:
It's fine.  It will end up populating the dask config variable `internal-inherit-config` automatically, but that seems fine to me.